python-sdk/example_dags/example_load_file.py

```diff
@@ -159,7 +159,7 @@
     # [START load_file_example_12]
     aql.load_file(
         input_file=File(
-            "s3://astro-sdk/python_sdk/example_dags/data/sample_pattern.csv",
+            "gs://astro-sdk/workspace/sample_pattern.csv",
```

Comment:
This should be changed to a pattern but I don't see a folder with pattern.

cc @utkarsharma2 Can you take care of this in a separate PR, please

Reply:
@kaxil https://github.com/astronomer/astro-sdk/pull/1497